CRITICAL - 2023-05-06 07:01:38 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 226.
 1 SYSTEMPATH\CodeIgniter.php(226): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(190): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(55): CodeIgniter\CodeIgniter->initialize()
ERROR - 2023-05-06 05:02:03 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'flatmates' in C:\xampp\htdocs\Flatmates\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\Flatmates\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'flatmates', 3306, '', 0)
#1 C:\xampp\htdocs\Flatmates\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\Flatmates\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\Flatmates\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\Flatmates\app\Models\dbmodels.php(69): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\Flatmates\app\Controllers\dbcontrollers.php(17): App\Models\Dbmodels->validate_ownerlogin_model(Array)
#6 C:\xampp\htdocs\Flatmates\system\CodeIgniter.php(934): App\Controllers\dbcontrollers->ownerlogin()
#7 C:\xampp\htdocs\Flatmates\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\dbcontrollers))
#8 C:\xampp\htdocs\Flatmates\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\Flatmates\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'flatmates' in C:\xampp\htdocs\Flatmates\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\Flatmates\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\Flatmates\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\Flatmates\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\Flatmates\app\Models\dbmodels.php(69): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\Flatmates\app\Controllers\dbcontrollers.php(17): App\Models\Dbmodels->validate_ownerlogin_model(Array)
#5 C:\xampp\htdocs\Flatmates\system\CodeIgniter.php(934): App\Controllers\dbcontrollers->ownerlogin()
#6 C:\xampp\htdocs\Flatmates\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\dbcontrollers))
#7 C:\xampp\htdocs\Flatmates\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\Flatmates\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2023-05-06 05:02:03 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'flatmates'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `owner_reg`
WHERE `email` = :email:
AND `password` = :password:', [...], false)
 3 APPPATH\Models\dbmodels.php(69): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\dbcontrollers.php(17): App\Models\Dbmodels->validate_ownerlogin_model([...])
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\dbcontrollers->ownerlogin()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\dbcontrollers))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-05-06 05:02:08 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'flatmates' in C:\xampp\htdocs\Flatmates\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\Flatmates\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'flatmates', 3306, '', 0)
#1 C:\xampp\htdocs\Flatmates\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\Flatmates\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\Flatmates\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\Flatmates\app\Models\dbmodels.php(69): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\Flatmates\app\Controllers\dbcontrollers.php(17): App\Models\Dbmodels->validate_ownerlogin_model(Array)
#6 C:\xampp\htdocs\Flatmates\system\CodeIgniter.php(934): App\Controllers\dbcontrollers->ownerlogin()
#7 C:\xampp\htdocs\Flatmates\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\dbcontrollers))
#8 C:\xampp\htdocs\Flatmates\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\Flatmates\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'flatmates' in C:\xampp\htdocs\Flatmates\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\Flatmates\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\Flatmates\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\Flatmates\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\Flatmates\app\Models\dbmodels.php(69): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\Flatmates\app\Controllers\dbcontrollers.php(17): App\Models\Dbmodels->validate_ownerlogin_model(Array)
#5 C:\xampp\htdocs\Flatmates\system\CodeIgniter.php(934): App\Controllers\dbcontrollers->ownerlogin()
#6 C:\xampp\htdocs\Flatmates\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\dbcontrollers))
#7 C:\xampp\htdocs\Flatmates\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\Flatmates\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2023-05-06 05:02:08 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'flatmates'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `owner_reg`
WHERE `email` = :email:
AND `password` = :password:', [...], false)
 3 APPPATH\Models\dbmodels.php(69): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\dbcontrollers.php(17): App\Models\Dbmodels->validate_ownerlogin_model([...])
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\dbcontrollers->ownerlogin()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\dbcontrollers))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-05-06 05:02:15 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'flatmates' in C:\xampp\htdocs\Flatmates\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\Flatmates\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'flatmates', 3306, '', 0)
#1 C:\xampp\htdocs\Flatmates\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\Flatmates\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\Flatmates\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\Flatmates\app\Models\dbmodels.php(69): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\Flatmates\app\Controllers\dbcontrollers.php(17): App\Models\Dbmodels->validate_ownerlogin_model(Array)
#6 C:\xampp\htdocs\Flatmates\system\CodeIgniter.php(934): App\Controllers\dbcontrollers->ownerlogin()
#7 C:\xampp\htdocs\Flatmates\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\dbcontrollers))
#8 C:\xampp\htdocs\Flatmates\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\Flatmates\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'flatmates' in C:\xampp\htdocs\Flatmates\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\Flatmates\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\Flatmates\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\Flatmates\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\Flatmates\app\Models\dbmodels.php(69): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\Flatmates\app\Controllers\dbcontrollers.php(17): App\Models\Dbmodels->validate_ownerlogin_model(Array)
#5 C:\xampp\htdocs\Flatmates\system\CodeIgniter.php(934): App\Controllers\dbcontrollers->ownerlogin()
#6 C:\xampp\htdocs\Flatmates\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\dbcontrollers))
#7 C:\xampp\htdocs\Flatmates\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\Flatmates\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2023-05-06 05:02:15 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'flatmates'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `owner_reg`
WHERE `email` = :email:
AND `password` = :password:', [...], false)
 3 APPPATH\Models\dbmodels.php(69): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\dbcontrollers.php(17): App\Models\Dbmodels->validate_ownerlogin_model([...])
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\dbcontrollers->ownerlogin()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\dbcontrollers))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-05-06 05:02:23 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'flatmates' in C:\xampp\htdocs\Flatmates\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\Flatmates\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'flatmates', 3306, '', 0)
#1 C:\xampp\htdocs\Flatmates\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\Flatmates\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\Flatmates\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\Flatmates\app\Models\dbmodels.php(69): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\Flatmates\app\Controllers\dbcontrollers.php(17): App\Models\Dbmodels->validate_ownerlogin_model(Array)
#6 C:\xampp\htdocs\Flatmates\system\CodeIgniter.php(934): App\Controllers\dbcontrollers->ownerlogin()
#7 C:\xampp\htdocs\Flatmates\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\dbcontrollers))
#8 C:\xampp\htdocs\Flatmates\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\Flatmates\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'flatmates' in C:\xampp\htdocs\Flatmates\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\Flatmates\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\Flatmates\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\Flatmates\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\Flatmates\app\Models\dbmodels.php(69): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\Flatmates\app\Controllers\dbcontrollers.php(17): App\Models\Dbmodels->validate_ownerlogin_model(Array)
#5 C:\xampp\htdocs\Flatmates\system\CodeIgniter.php(934): App\Controllers\dbcontrollers->ownerlogin()
#6 C:\xampp\htdocs\Flatmates\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\dbcontrollers))
#7 C:\xampp\htdocs\Flatmates\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\Flatmates\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2023-05-06 05:02:23 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'flatmates'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `owner_reg`
WHERE `email` = :email:
AND `password` = :password:', [...], false)
 3 APPPATH\Models\dbmodels.php(69): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\dbcontrollers.php(17): App\Models\Dbmodels->validate_ownerlogin_model([...])
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\dbcontrollers->ownerlogin()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\dbcontrollers))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-06 05:52:19 --> require(functions.php): Failed to open stream: No such file or directory
in APPPATH\Views\otp_verification.php on line 2.
 1 APPPATH\Views\otp_verification.php(2): CodeIgniter\Debug\Exceptions->errorHandler(2, 'require(functions.php): Failed to open stream: No such file or directory', 'APPPATH\\Views\\otp_verification.php', 2)
 2 APPPATH\Views\otp_verification.php(2): require()
 3 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\otp_verification.php')
 4 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('otp_verification', [], true)
 6 APPPATH\Controllers\Home.php(39): view('otp_verification')
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->otp_verification()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-06 05:52:25 --> require(functions.php): Failed to open stream: No such file or directory
in APPPATH\Views\otp_verification.php on line 2.
 1 APPPATH\Views\otp_verification.php(2): CodeIgniter\Debug\Exceptions->errorHandler(2, 'require(functions.php): Failed to open stream: No such file or directory', 'APPPATH\\Views\\otp_verification.php', 2)
 2 APPPATH\Views\otp_verification.php(2): require()
 3 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\otp_verification.php')
 4 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('otp_verification', [], true)
 6 APPPATH\Controllers\Home.php(39): view('otp_verification')
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->otp_verification()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-06 05:52:32 --> require(functions.php): Failed to open stream: No such file or directory
in APPPATH\Views\otp_verification.php on line 2.
 1 APPPATH\Views\otp_verification.php(2): CodeIgniter\Debug\Exceptions->errorHandler(2, 'require(functions.php): Failed to open stream: No such file or directory', 'APPPATH\\Views\\otp_verification.php', 2)
 2 APPPATH\Views\otp_verification.php(2): require()
 3 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\otp_verification.php')
 4 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('otp_verification', [], true)
 6 APPPATH\Controllers\Home.php(39): view('otp_verification')
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->otp_verification()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-06 05:52:41 --> Call to undefined function hai()
in APPPATH\Views\otp_verification.php on line 23.
 1 SYSTEMPATH\View\View.php(213): include()
 2 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('otp_verification', [], true)
 4 APPPATH\Controllers\Home.php(39): view('otp_verification')
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->otp_verification()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-06 05:53:37 --> require_once(functions.php): Failed to open stream: No such file or directory
in APPPATH\Views\otp_verification.php on line 2.
 1 APPPATH\Views\otp_verification.php(2): CodeIgniter\Debug\Exceptions->errorHandler(2, 'require_once(functions.php): Failed to open stream: No such file or directory', 'APPPATH\\Views\\otp_verification.php', 2)
 2 APPPATH\Views\otp_verification.php(2): require_once()
 3 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\otp_verification.php')
 4 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('otp_verification', [], true)
 6 APPPATH\Controllers\Home.php(39): view('otp_verification')
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->otp_verification()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
